specs/v2-sketch.yaml

```diff
@@ -282,40 +264,29 @@ components:
               student:
                 type: boolean
                 default: false
-    Organization:
-      description: "NOTE: The definition of Organization is subject to change."
-      type: object
-      properties:
-        id:
-          type: string
-          format: uuid
     Project:
       description: "NOTE: The definition of Project is subject to change."
       type: object
+      required:
+        - id
+        - vcs_url
       properties:
         id:
+          description: Project ID in the form `vcs-slug/org-name/repo-name`
           type: string
-          format: uuid
-        vcs_slug:
-          type: string
-          enum: ["github", "bitbucket"]
-        name:
-          type: string
-          example: "api-preview-docs"
-        organization_id:
-          type: string
-          format: uuid
-        organization_name:
+          example: "bitbucket/CircleCI-Public/api-preview-docs"
```

Comment:
Let's go with "gh" here

Reply:
If we commit to `gh` and `bb` then we need to allow those aliases in the 1.1 API. It's a bit of extra work but not a blocker.

Reply:
I think we should make that commitment -- perhaps even support both in both versions, but definitely do the aliasing in 1.1